src/DIRAC/FrameworkSystem/Utilities/diracx.py

```diff
@@ -104,3 +104,20 @@ def TheImpersonator(credDict: dict[str, Any], *, source: str = "") -> Generator[
         client.__enter__()
         diracx_client_cache[token_location] = client
     yield client
+
+
+def diracxVerifyConfig(cfgData):
```

Comment:
This won't work for converting the CS, see https://diracx.diracgrid.org/en/latest/admin/how-to/install/convert-cs/ for the procedure.

Running it in a subproces with a timeout is probably a good idea.